src/Analysis/Codelyzer.Analysis.Common/AnalyzerConfiguration.cs

```diff
@@ -49,6 +49,7 @@ public class MetaDataSettings
         public bool InterfaceDeclarations = false;
         public bool EnumDeclarations = false;
         public bool StructDeclarations = false;
+        public bool ReturnStatements = true;
```

Comment:
Default should not be true. Please set this using the same logic for Annotations, Enums, or structs. We set this to true in the Cli if DetailedAnalysis flag is set.

Reply:
Yep... realized this should be false after pushing it out. Should be updated now